.travis.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
I just decided to try out adding support for Python v3.8 looks like we need to compile our own `libgit` for Travis CI to work but now we're successfully testing `battenberg` on 3.6, 3.7 & 3.8 😄